docs/guides/export.md

```diff
@@ -0,0 +1,120 @@
+    ```
+
+## Requirements.txt format
+
+The `requirements.txt` format is the most widely supported format for Python dependencies. It can be used with `pip` and other Python package managers.
+
+### Basic usage
+
+```console
+$ uv export --format requirements.txt
+```
+
+The generated `requirements.txt` file can then be installed via `uv pip install`, or with other tools like `pip`.
+
+!!! note
+
+    In general, we recommend against using both a `uv.lock` and a `requirements.txt` file. The
+    `uv.lock` format is more powerful and includes features that cannot be expressed in
+    `requirements.txt`. If you find yourself exporting a `uv.lock` file, consider opening an issue
+    to discuss your use case.
+
+## Pylock.toml format (PEP 751)
+
+[PEP 751](https://peps.python.org/pep-0751/) defines a TOML-based lockfile format for Python dependencies. uv can export your project's dependency lockfile to this format.
+
+### Basic usage
+
+```console
+$ uv export --format pylock.toml
+```
+
+## CycloneDX SBOM format
+
+uv can export your project's dependency lockfile as a Software Bill of Materials (SBOM) in CycloneDX format. SBOMs provide a comprehensive inventory of all software components in your application, which is useful for security auditing, compliance, and supply chain transparency.
+
+### What is CycloneDX?
+
+[CycloneDX](https://cyclonedx.org/) is an industry-standard format for creating Software Bill of Materials. CycloneDX is machine readable and widely supported by security scanning tools, vulnerability databases, and Software Composition Analysis (SCA) platforms.
+
+### Basic usage
+
+To export your project's lockfile as a CycloneDX SBOM:
+
+```console
+$ uv export --format cyclonedx1.5
+```
+
+This will generate a CycloneDX v1.5 JSON containing your project and all of its dependencies.
+
+### SBOM Structure
+
+The generated SBOM follows the [CycloneDX specification](https://cyclonedx.org/specification/overview/) and includes:
+
+#### Metadata
+
+- **Timestamp**: When the SBOM was generated
+- **Tools**: Information about uv (vendor, name, version)
+- **Root component**: Your project as the main application component
+- **Serial number**: A unique identifier for the SBOM
+- **Spec version**: The version of the CycloneDX specification
+
+#### Components
+
+Each dependency is represented as a component with:
+
+- **Type**: `library` for external packages, `application` for your project and workspace members
+- **Name and version**: Package identifier
+- **PURL**: Package URL for registry packages (e.g., `pkg:pypi/requests@2.31.0`)
+- **Properties**: Custom metadata including:
+  - `uv:package:marker`: Environment markers (e.g., `python_version >= "3.8"`)
+  - `uv:workspace:path`: Relative path for workspace members
+
+#### Dependencies
+
+A complete graph showing which components depend on which other components. This allows tools to trace vulnerability impact through the dependency tree.
```

Comment:
As a high-level thought here: I _think_ we could probably keep this section simpler by linking to the CycloneDX for most of this (or a nice generic guide, if one exists?) and then limiting the content here to the parts that are specific to uv (like the `uv:*` properties).